src/openforms/js/components/admin/forms/ComponentSelection.stories.mdx

```diff
@@ -0,0 +1,40 @@
+import {ArgsTable, Canvas, Meta, Story} from '@storybook/addon-docs';
+
+import {FormContext} from 'components/admin/form_design/Context';
+
+import ComponentSelection from './ComponentSelection';
+
+<Meta title="Django admin/ComponentSelection" component={ComponentSelection} />
+
+# Field
+
+Select widget with components as options
```

Comment:
I'm not sure if this component should be part of the generic Django admin components, because it uses `FormContext` and seems rather OF specific

Reply:
yeah, it's form-designer specific, so grouping it in there would make more sense